doc/user_documentation.rst

```diff
@@ -293,6 +295,24 @@ of the names ``<coprname>:pr:<pr_id>``. ``<pr_id>`` is ID of the pull request op
 
     $ sudo dnf copr enable <ownername>/<coprname>:pr:<pr_id>
 
+Create build from PR comment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+How to set up your project in Pagure:
+
+- Go to ``Settings`` on your project
+- Go to ``Project Options`` section
+- Select ``Fedmsg notifications`` option
+
+How to se up project in Copr:
+
+- Create new package in your Copr repository
+- Select ``SCM`` source type and specify ``Clone url``
+- Check the ``Auto-rebuild`` section
+
+From now on, everytime you comment ``[copr-build]`` keyword into PR, Copr build is automatically triggered into
+the specified project in Copr.
```

Comment:
Free form (please update): From now on, you get automatic rebuilds for new pull-requests (including force pushes), you can drop a comment with `[copr-build]` keyword to re-trigger the build(s), and copr also automatically builds on pushes (commit) to the Pagure repo.

Reply:
this describes impact from auto-rebuild and PR command rebuild together but as I described in https://github.com/fedora-copr/copr/pull/2662#discussion_r1167479095 these two really differs in the settings to enable. Seems like there were some changes in Pagure recently

Reply:
I still don't get how these two are actually different :-) I likely miss something and I need yet another video-meeting iteration.  The point of this PR should be to have a self-contained single howto (not two howtos).